docs/physics/gammaray_deposition.ipynb

```diff
@@ -0,0 +1,573 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    sum(escape_energy) / (sum(escape_energy) + sum(ejecta_energy))
</pre><p>we should use <code>np.sum</code>  and some description</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1698/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>